src/patch_parser.py

```diff
@@ -309,6 +309,7 @@ def find_comments(parent_lines: List[Line], all_child_lines: List[Line]) -> List
     return merge_comment_lines(comment_lines)
 
 def diff_reply(parent: Message, child: Message) -> List[Comment]:
+    #TODO: Potential bug, to_lines takes a str, but Message.content is a List
```

Comment:
Well, is it a bug? Have you run this code?

Reply:
the thing is, it works? like comments are uploaded? but sometimes it throws an error?

Reply:
like occasionally, there will be an error 'List does not have method splitlines()' or something along those lines

Reply:
Ok, so sounds like we have a random type error where we sometimes assign a str to .content.
I'm fine with leaving this as a TODO here, but perhaps reword to

```suggestion
    #TODO: to_lines only works on str, but Message.content is also sometimes a List.
```

Reply:
SGTM.